### PR TITLE
Implementation mongodb

### PR DESCRIPTION
**Pull request by DilanOjeda**
Please check!

### CODE REVIEW

**Review by stospina — COMMENTED** [review]
Everything looks good...

File: CONVERTER-SERVICE/src/app.js [hunk 1]
```diff
@@ -13,10 +13,17 @@ with Jalasoft.
...
-
+const Database = require('./database/database.db');
+require('./models/file.model');
 const app = express();
 dotenv.config();
 
+const database = new Database(process.env.MONGO_URI, process.env.MONGO_DB);
+
+database.connectDatabase()
+  .then(() => console.info('Database is connected'))
```
Comment (luisdavidparra) [inline CONVERTER-SERVICE/src/app.js]: We shouldn't use console.info or console.log in the final code.
And the structure of .then and .chatch we are using is in block not inline.

File: CONVERTER-SERVICE/src/database/database.db.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  // Connects to the data base
+  async connectDatabase() {
+    try {
+      await mongoose.connect( this._uri + this.database, {
+          useNewUrlParser: true,
+          useUnifiedTopology: true,
+      });
+    } catch (error) {
+        console.error();( 'Error: ', error );
```
Comment (luisdavidparra) [inline CONVERTER-SERVICE/src/database/database.db.js]: In this case you're already handling the error in the throw, is not necessary to console.error.

File: CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            .then(() => {
+              return next();
+            })
+            .catch(() => {
+              res.json({ error: 'Something went wrong when saving in dabatase' });
+            })
+        }
+      });
+    } catch (error) {
+      req.json({ error: 'Something went wrong when uploading file'});
```
Comment (luisdavidparra) [inline CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js]: This have to be `res.json` not `req.json`

**Review by luisdavidparra — CHANGES_REQUESTED** [review]

File: CONVERTER-SERVICE/src/app.js (hunk 1 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/app.js]: It's already done

File: CONVERTER-SERVICE/src/database/database.db.js (hunk 2 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/database/database.db.js]: It's already done

File: CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js (hunk 3 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js]: It's already done

File: CONVERTER-SERVICE/src/database/database.db.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    try {
+      await mongoose.connect( this._uri + this.database, {
+          useNewUrlParser: true,
+          useUnifiedTopology: true,
+      });
+    } catch (error) {
+        throw new Error( 'There was an connection error.' );
+    }
+  }
+
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/database/database.db.js]: This line is unnecesary

**Review by florpadilla — COMMENTED** [review]
It looks good. Just delete a line at database.db.js

File: CONVERTER-SERVICE/src/database/database.db.js (hunk 4 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/database/database.db.js]: It's already done

**Review by luisdavidparra — APPROVED** [review]

**Pull request merged**